docs/src/pages/react-native.md

```diff
@@ -3,8 +3,78 @@ id: react-native
 title: React Native
 ---
 
-React Query is designed to work out of the box with React Native, with the exception of the devtools, which are only supported with React DOM at this time. 
+React Query is designed to work out of the box with React Native, with the exception of the devtools, which are only supported with React DOM at this time.
 
 There is a 3rd party [Flipper](https://fbflipper.com/docs/getting-started/react-native/) plugin which you can try: https://github.com/bgaleotti/react-query-native-devtools
 
 If you would like to help us make the built-in devtools platform agnostic, please let us know!
+
+## Online status management
+
+React Query already supports auto refetch on reconnect in web browser.
+To add this behavior in React Native you have to use React Query `onlineManager` as in the example below:
+
+```ts
+import React from 'react'
+import NetInfo from '@react-native-community/netinfo'
+import { onlineManager } from 'react-query'
+import { Platform } from 'react-native'
+
+export function useOnlineManager() {
+  React.useEffect(() => {
+    if (Platform.OS !== 'web') {
+      return NetInfo.addEventListener(state => {
```

Comment:
hmm, the [onlinemanager docs](https://react-query.tanstack.com/reference/onlineManager#onlinemanagerseteventlistener) have a slightly different example:

```
 import NetInfo from '@react-native-community/netinfo'
 import { onlineManager } from 'react-query'
 
 onlineManager.setEventListener(setOnline => {
   return NetInfo.addEventListener(state => {
     setOnline(state.isConnected)
   })
 })
```

Reply:
Agree, we can keep the same example.